shelly/src/main/java/au/com/jtribe/shelly/Email.java

```diff
@@ -130,7 +129,7 @@ public boolean send() {
         }
 
         if (emailIntent.resolveActivity(this.context.getPackageManager()) != null) {
-            this.context.startActivity(emailIntent);
+            this.context.startActivity(Intent.createChooser(emailIntent,"Share email via:"));
```

Comment:
could do something like

    public boolean send() {
     return launch(null);
    }

    public boolean send(@NonNull String title) {
     return launch(title);
    }

    private boolean launch(String title) {
    //old logic in here, if title != null then use the title for createChooser
    }